src/ray/object_manager/object_manager.h

```diff
@@ -267,6 +267,10 @@ class ObjectManager : public ObjectManagerInterface {
   /// Register object remove with directory.
   void NotifyDirectoryObjectDeleted(const ObjectID &object_id);
 
+  /// Handle any push requests that were made before an object was available.
+  /// This is invoked when an "object added" notification is received from the store.
+  void HandleUnfulfilledPushRequests(const ObjectInfoT &object_info);
```

Comment:
This seems unrelated to the driver table and to cleaning up the GCS, is that right?

Same with all of the object manager changes.

Reply:
Yes, it's unrelated. This just improves the readability of some (at the time) recent changes to the object manager. We could make this a separate PR, but it's a relatively benign change.